tests/flow/test_cms.py

```diff
@@ -107,6 +114,30 @@ def test_incrby_query(self):
         #     self.assertEqual([1], self.cmd('cms.query', 'test', 'bar'))
         #     self.assertEqual([0], self.cmd('cms.query', 'test', 'nonexist'))
 
+    def test_batchquery(self):
+        self.assertOk(self.cmd('cms.initbydim', 'k1', '100', '5'))
+        self.assertOk(self.cmd('cms.initbydim', 'k2', '100', '5'))
+        self.assertOk(self.cmd('cms.initbydim', 'k3', '100', '6'))
+        self.assertEqual([5], self.cmd('cms.incrby', 'k1', 'v1', '5'))
+        self.assertEqual([5], self.cmd('cms.incrby', 'k2', 'v2', '5'))
+        self.assertEqual([5], self.cmd('cms.incrby', 'k3', 'v3', '5'))
+        self.assertEqual([5, 5], self.cmd('cms.batchquery', 'KEYS', 'k1', 'k2', 'VALUES', 'v1', 'v2'))
```

Comment:
Can we plese have an example with a count from two keys?

Reply:
`self.assertEqual([5, 5], self.cmd('cms.batchquery', 'KEYS', 'k1', 'k2', 'VALUES', 'v1', 'v2'))`
I think it is want you want.

Reply:
@JinxLbj Let's add  
```
self.assertEqual([5], self.cmd('cms.incrby', 'k1', 'v2', '5'))
```
so the batch query will return [5, 10]